app/services/recommendations_service/media.rb

```diff
@@ -25,6 +25,18 @@ def recommendations_for(klass)
       reccs.map(&:first)
     end
 
+    def realtime_recommendations_for(klass)
+      res = client.get("realtime/#{user.id}/")
+      reccs = res.body.dig('results', klass.name)
+      reccs = reccs.reduce(&:merge)
+      media_ids = reccs.keys.map { |key| key.split(':').last.to_i }
+      media = klass.find(media_ids).index_by { |m| "#{m.class.name}:#{m.id}" }
```

Comment:
Would probably be good to split this out into an `enrich` method, so that the logic for swapping `"Anime:321"` for actual `Anime` instances can be centralized